src/dash-table/handlers/cellEvents.ts

```diff
@@ -1,57 +1,69 @@
-import * as R from 'ramda';
-import { SelectedCells, ICellFactoryProps } from 'dash-table/components/Table/props';
+import { min, max, set, lensPath } from 'ramda';
```

Comment:
I made a couple of forays into specific ramda imports - my local linter complains about `* as R` and as @chriddyp pointed out (somewhere?) we may get a lighter build that way too.

Reply:
Yes. Depending on your linter, it may or may not handle the typescript option for `* as R`.
The way imports are used here webpack should prune the unused content automagically but this is fine too.